### DISCUSSION TITLE
Does OpenTelemetry support error tracking like Sentry?

### DISCUSSION THREAD

**Discussion by simlarsen** [open-telemetry/opentelemetry-specification#3911] [Q&A]
I'm core contributor to OneUptime (https://oneuptime.com). I'm working on Otel Backend. Does Otel does Error Tracking as well? Like Sentry? If yes, can you please point me to a documentation? If no, is there a plan in the future to support it?

**Comment by svrnm** [comment]
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/exceptions.md might be what you are looking for. Not sure what sentry does or doesn't but this might be a starting point for you.

**Comment by reyang** [comment]
@simlarsen Please use Discussions or raising the question on our Slack channel.
